### PR TITLE
Update BTCPayServer project description and summary

### PR DESCRIPTION
**Pull request by lucasmogu**
Update "summary" and "description" as per OpenSats/volunteers#1

### CODE REVIEW

File: data/projects/btcpayserver.mdx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: 'BTCPay Server'
 dateAdded: '2022-09-22'
-summary: 'Free, open-source & self-hosted bitcoin payment gateway that allows self-sovereign individuals and businesses to accept bitcoin payments online or in person without any fees.'
+summary: 'Start accepting Bitcoin payments with 0% fees & no third-party.'
```
Comment (Arvin21M) [inline data/projects/btcpayserver.mdx:4]:
```suggestion
summary: 'Start accepting Bitcoin payments with 0% processing fees & no third-party intermediary.'
```

File: data/projects/btcpayserver.mdx [hunk 2]
```diff
@@ -15,4 +15,4 @@ tags: ['Bitcoin', 'Lightning', 'Wallet', 'Node', 'Commerce', 'Infrastructure']
 
 ## About this project
 
-BTCPay Server provides numerous features for self-sovereign individuals and businesses: Direct, peer-to-peer Bitcoin payments | No transaction fees (other than the network fee) | No fees, middleman or KYC | Non-custodial (complete control over the private key) | Lightning Network support (LND, c-lightning, Eclair, and Ptarmigan) | Tor support | Share your instance with friends (multi-tenant) | Invoice management and Payment requests | Apps: Point of sale, crowdfunding, donation button | Full-node reliant wallet with hardware wallet integration and SegWit support | Payjoin Support
+BTCPay Server is a self-hosted, open-source cryptocurrency payment processor. It's secure, private, censorship-resistant, and free. Simply launch a server (cloud, your own hardware device, or an existing host), connect your e-commerce store or one of many apps, and funds arrive directly to your connected wallet. Simple to use with native wallet management, and great for invoicing and accounting.
```
Comment (Arvin21M) [inline data/projects/btcpayserver.mdx:18]:
```suggestion
BTCPay Server is a self-hosted, open-source cryptocurrency payment processor. It's secure, private, censorship-resistant, and free. Simply launch a server (cloud, your own hardware device, or an existing host), connect your e-commerce store or one of the many apps, and funds arrive directly to your connected wallet. Ability to also facilitate cross-border transactions without conversion fees. Easy to use with native wallet management, and great for invoicing and accounting.
```

**Review by Arvin21M — CHANGES_REQUESTED** [review]
@lucasmogu Just some light touch ups. Line 18: I changed **Simple** to **Easy** because you used _Simply_ earlier in the paragraph and I was compelled to request the change, lol. Thank you!

**Comment by lucasmogu** [comment]
@Arvin21M thanks for the revision, all suggestions for all projects have been committed now. Great to see our GitHub skills improving :)

**Review by dergigi — APPROVED** [review]
Looks good to me ✅

**Pull request merged**